### DISCUSSION TITLE
[1] Dinder - Projektstart

### DISCUSSION THREAD

**Discussion by Manuel-F-04** [dhbw-ka-tinf22b5-dinder/Dinder#4] [Projektblog]
### Teammitglieder:
Das Team besteht aus folgenden Teammitgliedern: 
 - Jannis Fuchs (Frontend)
 - Lukas Schulz (Frontend) 
 - Jonathan Schäfer (Backend)
 - Tobias Raible (Backend)
 - Manuel Franz (Database)

### Produktvision:
Wir haben uns für eine Webanwendung entschieden, welche Dienstleistungen vermitteln soll. Unser Projekt heißt Dinder, also Tinder für Dienstleistungen.

### Techstack:
Wir beabsichtigen folgenden Teck-Stack zu verwenden:  
 - Web-App
 - Frontend: evtl. Angular oder React
 - Backend: Spring Boot
 - Datenbank: MariaDB
 - IDE: IntelliJ, teils Eclipse

### Scrum-Setup:
 - YouTrack
 - Sprintlänge: 2 Wochen
 - Daily: Dienstag nach der Vorlesung 15 min
 - Retrospektive: Nach 2 Wochen (Nach einem Sprint)
 - Planning: Alle 2 Wochen Dienstag

### Sprintbacklog - in Form des Screenshots des Kanbanboards
![grafik](https://github.com/dhbw-ka-tinf22b5-dinder/Dinder/assets/99080173/2895e7a2-eb43-4108-8de3-e59d8e91e00c)

Anmerkung: Die Tasks, bei denen keine Person zugewiesen ist, soll von allen Beteiligten absolviert werden (hauptsächlich Information über Plattformen und Software).

**Comment by ghost** [comment]
Sieht sehr gut aus, vor allem der Tech Stack. Mich wundert nur ein bisschen, dass ihr nur ein Daily pro Woche habt?

**Reply by Manuel-F-04** [reply]
Da wir keine ganzen Arbeitstage haben (nur ein paar Stunden pro Person pro Woche) bietet sich für uns ein wöchentliches Daily an, um das Projekt nicht unnötig organisatorisch aufzublähen.

**Comment by Puggingtons** [comment]
Sehr kreativer Name! Für euer Frontend würde ich empfehlen, dass ihr euch direkt entscheidet, welches ihr nehmt. In dem Sprintbacklog wird ja auch schon Angular erwähnt
